search-app/test/cmr/search/test/results_handlers/subscriptions_json_results_handler.clj

```diff
@@ -0,0 +1,26 @@
+(ns cmr.search.test.results-handlers.subscriptions-json-results-handler
```

Comment:
Not see much value in this test. Delete it.

Reply:
The unit level tests are good for development and without needing the entire system running.

Reply:
This test basically tests that I used these keys to construct a map and verify that these keys exist in the map that is returned. It is too trivial. And these keys in the map is in certain format does not guarantee that the final JSON response key in that format. That is why I recommended doing the check in the system-int-test rather than in the unit test. If there is test in system-int-test, this unit test will become even more unnecessary. Only when we don't add the verification of key format in the system-int-test, does this test have some value.

Reply:
System integration tests do not count toward code coverage. Unit tests do, this assists with maintaining required code coverage levels.

Reply:
If our goal of higher code coverage is to have a higher code coverage number, I have nothing. :)